packages/BetterPhpDocParser/Printer/PhpDocInfoPrinter.php

```diff
@@ -55,7 +55,7 @@ final class PhpDocInfoPrinter
     /**
      * @var string
```

Comment:
```suggestion
     * @var string Uses a hardcoded unix-newline since most codes use it (even on windows) - otherwise we would need to normalize newlines 
```